src/commands/project/pipeline.ts

```diff
@@ -97,13 +133,7 @@ export const fetchValidateValues = (
     );
   }
   const azureDevops = spkConfig?.azure_devops;
-  if (!opts.repoUrl) {
-    throw buildError(
-      errorStatusCode.VALIDATION_ERR,
-      "project-pipeline-err-repo-url-undefined"
-    );
-  }
```

Comment:
can you help to remove `project-pipeline-err-repo-url-undefined` from src/lib/i18n.json? thanks

Reply:
is there a reason why we are removing it?

Reply:
Got it, will do.